xssp_rest/tasks.py

```diff
@@ -252,10 +260,9 @@ def get_task(input_type, output_type):
         if output_type == 'hssp_hssp' or \
            output_type == 'hssp_stockholm':
```

Comment:
Doesn't the type `hg_hssp` need to be added here? Have you tested this by running it in the browser?